Makefile.amiga

```diff
@@ -0,0 +1,65 @@
+# Usage:
```

Comment:
Please add the name of this file to the `EXTRA_DIST = ` list in `Makefile.am`.

Reply:
Done!